File: messages/src/main/java/com/thundashop/core/pmsmanager/PmsOrderStatsFilter.java
package com.thundashop.core.pmsmanager;

import com.thundashop.core.common.DataCommon;
import java.io.Serializable;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class PmsOrderStatsFilter extends DataCommon {
    public String name = "";
    public Date start;
    public Date end;
    List<PmsPaymentMethods> methods = new ArrayList();
    public String displayType;
    public String priceType;
    public boolean includeVirtual = false;
    public boolean fromPmsModule = false;
    public Integer shiftHours = 0;
    
    public String savedPaymentMethod = "";
    public String channel;
}

File: html5apps/terminal/events/core/pmsmanager/PmsOrderStatsFilter.php
<?php
class core_pmsmanager_PmsOrderStatsFilter extends core_common_DataCommon  {
	/** @var String */
	public $name;

	/** @var String */
	public $start;

	/** @var String */
	public $end;

	/** @var core_pmsmanager_PmsPaymentMethods[] */
	public $methods;

	/** @var String */
	public $displayType;

	/** @var String */
	public $priceType;

	/** @var String */
	public $includeVirtual;

	/** @var String */
	public $shiftHours;

	/** @var String */
	public $savedPaymentMethod;

	/** @var String */
	public $channel;

}
?>